reconcile/database_access_manager.py

```diff
@@ -439,42 +442,52 @@ def _create_database_connection_parameter(
     oc: OCClient,
     admin_secret_name: str,
     user_secret_name: str,
-) -> DatabaseConnectionParameters:
+) -> dict[str, DatabaseConnectionParameters]:
     def _decode_secret_value(value: str) -> str:
         return base64.b64decode(value).decode("utf-8")
 
+    print("creating database connection parameter")
```

Comment:
print? shouldn't this be done via `logging`?

Reply:
:broom: